package.json

```diff
@@ -4,7 +4,8 @@
   "description": "Proxies nodejs require in order to allow overriding dependencies during testing.",
   "main": "index.js",
   "scripts": {
-    "test": "mocha"
+    "test": "mocha",
+    "lint": "jshint index.js lib"
```

Comment:
Hey please don't do this, contributing a linter in an unrelated PR is quite unwelcome

Reply:
Oh, I noticed there was a `.jshintrc` in the root folder.  :P  I'll pull it out.

Reply:
Yeah yeah my fault I didn't even realize there was jshintrc stuff in there. Since it hasn't been in CI for a while I'm just going to replace it with something a bit more ambitious/opinionated that's actually catching bugs.